.github/workflows/nightly-upstream-snapshot-build.yml

```
@@ -73,19 +73,27 @@ jobs:
       - name: Set up Docker Buildx
         uses: docker/setup-buildx-action@v3
 
+      - name: Set current date as env variable
+        id: generate_time_stamp
+        run: echo "nowTimeTag=$(date +'%Y-%m-%dT%H-%M-%S')" >> $$GITHUB_OUTPUT
+
+      - name: Set image name to output
+        id: imageNameOutput
+        run: echo "rcImage=${{ env.IMAGE_NAME }}:${{steps.generate_time_stamp.outputs.nowTimeTag}}" >> "$GITHUB_OUTPUT"
```

nit: Don't need quotation marks around `$GITHUB_OUTPUT`, doesn't match the rest of the workflow
```suggestion
        run: echo "rcImage=${{ env.IMAGE_NAME }}:${{steps.generate_time_stamp.outputs.nowTimeTag}}" >> $GITHUB_OUTPUT
```

---

Thanks, updated.